### PR TITLE
Feature/245/create acct name field update

### PR DESCRIPTION
**Pull request by Datguyovrder**
### Describe The Problem Being Solved:
When a user click on the first and last name box in create an account, the blue text in the respective boxes say "First" and "Last". I changed it so that it reads "First Name" and "Last Name".

Screen Shot 2019-05-15 at 7 24 00 PM

Screen Shot 2019-05-15 at 7 34 09 PM

### Checklist For Submitter

* [X] I have documented all new functionality
* [X] I have screenshotted new UI changes and attached them to this PR

### CODE REVIEW

File: frontend/src/components/SignUp/SignUp.js [hunk 1]
```diff
@@ -60,11 +60,12 @@ class SignUp extends Component {
                   name="firstName"
                   className="sign-in-input firstName col-md form-control-lg"
                   placeholder="First"
+                  required
```
Comment (SamDesai333) [inline frontend/src/components/SignUp/SignUp.js]: Good job adding in required to all the necessary fields for validation!

File: frontend/src/components/SignUp/SignUp.js [hunk 2]
```diff
@@ -60,11 +60,12 @@ class SignUp extends Component {
...
                   onFocus={event => (event.target.setAttribute('placeholder', ''))}
                   onBlur={event => (event.target.setAttribute('placeholder', 'First'))}
```
Comment (vhoof) [inline frontend/src/components/SignUp/SignUp.js]: Should we change these placeholders, as well?

Comment (tkhutch) [reply frontend/src/components/SignUp/SignUp.js]: I know it's weird, but the design doc has "First" and "Last" as the placeholders and "First Name" and "Last Name" as the popup text. No changes are needed to that.

Comment (tkhutch) [reply frontend/src/components/SignUp/SignUp.js]: @vhoof If you have questions about the design/product, please mention me as I usually assume questions are for the devs

Comment (vhoof) [reply frontend/src/components/SignUp/SignUp.js]: It was intended for him, but that's cool.

**Pull request merged**